content/source/docs/enterprise/api/team-access.html.md

```diff
@@ -69,17 +71,22 @@ $ curl \
 }
 ```
 
-## Add Team access to a Workspace
+## Add Team Access to a Workspace
```

Comment:
I notice the capitalisation here, should that be documented in the template? Would it be better to not capitalise to prevent differences?

Reply:
That's just part of the general style of making headers title-case.

Reply:
:+1: